src/environment/shell.go

```diff
@@ -500,22 +501,29 @@ func (env *ShellEnvironment) RunShellCommand(shell, command string) string {
 	return ""
 }
 
-func (env *ShellEnvironment) HasCommand(command string) bool {
+func (env *ShellEnvironment) HasCommandPath(command string) string {
```

Comment:
This implies a boolean return but isn't, can we rename this to `CommandPath`?

Reply:
renamed